pyflakes/checker.py

```diff
@@ -1501,9 +1493,9 @@ def SUBSCRIPT(self, node):
                     self.handleNode(slice_children[0], node.slice)
                 # Only the first argument of an Annotated type will always
                 # correspond to an actual type.
-                with self._enter_typing_annotated():
-                    for slice_child in slice_children[1:]:
-                        self.handleNode(slice_child, node)
+                self._in_annotation = False
```

Comment:
I believe this needs to be restored afterwards (as in a context manager that does the opposite of the one which enables `_in_annotation`)

for example this should fail (and may not with this change?):

```python
def f(x: Union[Annotated['int', '>0'], 'integer']) -> ...:
```

Reply:
You are right, that test case failed. I added it as a test case and fixed the code.